src/components/__tests__/FeedbackBadge.spec.tsx

```diff
@@ -0,0 +1,94 @@
+import { waitFor } from '@testing-library/react-native'
+import { mocked } from 'jest-mock'
+import React from 'react'
+
+import { SCORE_THRESHOLD_POSITIVE_FEEDBACK, EXERCISE_FEEDBACK } from '../../constants/data'
+import { useLoadAsync } from '../../hooks/useLoadAsync'
+import { getLabels } from '../../services/helpers'
+import render from '../../testing/render'
+import FeedbackBadge from '../FeedbackBadge'
+
+jest.mock('@react-navigation/native')
+jest.mock('../../hooks/useLoadAsync', () => ({
+  useLoadAsync: jest.fn(),
+}))
+
+const mockSetFeedback = jest.fn()
+
+describe('FeedbackBadge', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  const renderFeedbackBadge = (levelIdentifier: { disciplineId: number; level: number } | null) =>
+    render(<FeedbackBadge levelIdentifier={levelIdentifier} setFeedback={mockSetFeedback} />)
+
+  it('should not show when no level info is provided', async () => {
+    mocked(useLoadAsync).mockImplementation(() => ({
```

Comment:
you can use "mockData" function from `mockUseLoadFromEndpoint.ts`
Same for other tests in this file.